File: Talleres-2B/Taller-3/Ejercicio3.py
print("\t----SUMA DE DOS MATRICES A Y B EN UNA MATRIZ C----")

n = int(input("Ingrese el tamaño de las matrices a ingresar: "))

#Ingreso de datos en las matrices
A = []

print("\nIngresar los elementos de la matriz A")
for i in range (n):
    A.append([])
    for j in range (n):
        x = int(input("A["+str(i)+"]["+str(j)+"]: "))
        A[i].append(x)

B = []
print("\nIngresar los elementos de la matriz B")
for i in range (n):
    B.append([])
    for j in range (n):
        x = int(input("B["+str(i)+"]["+str(j)+"]: "))
        B[i].append(x)

print("\nLa matriz A es: ")
for i in range (n):
    for j in range (n):
        print(A[i][j], end="   ")
    print("")

print("\nLa matriz B es: ")
for i in range (n):
    for j in range (n):
        print(B[i][j], end="   ")
    print("")

# Suma de matrices
C = []

for i in range (n):
    C.append([])
    for j in range (n):
        x = A[i][j] + B[i][j]
        C[i].append(x)

print("\nLa matriz C es: ")
for i in range (0, n):
    for j in range (0, n):
        print(C[i][j], end="   ")
    print("")

File: Talleres-2B/Taller-3/OperacionesMatrices/src/operacionesmatrices/Ejercicio3.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package operacionesmatrices;
import java.util.Scanner;
/**
 *
 * @author SALAA
 */
public class Ejercicio3 {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        System.out.println("\t----SUMA DE DOS MATRICES A Y B EN UNA MATRIZ C----");
        //Declaración e Inicialización de las variables
        int n, i, j;
        n = 0;
        //Ingreso de datos en la matriz 
        System.out.println("Ingrese el tamaño de las matrices: ");
        n = leer. nextInt();
        
        System.out.println("Sus matrices A y B corresponden a un tamaño de: " +n +" X " +n);
        
        int [][] A = new int [n][n];
        System.out.println("\nIngrese los elementos de la matriz A: ");
        for (i=0; i<n; i++){
            for (j = 0; j<n; j++){
                System.out.println("A[" +i +"] [" +j +"] =");
                A[i][j] = leer.nextInt();
            }
        }
        System.out.println("\nLa matriz A es: ");
        for (i = 0; i<n; i++){
            for ( j= 0; j<n; j++){
                System.out.print(A[i][j] +"\t");
            }
            System.out.println("");
        }
        int [][] B = new int [n][n];
        System.out.println("\nIngrese los elementos de la matriz B: ");
        for (i=0; i<n; i++){
            for (j = 0; j<n; j++){
                System.out.println("B[" +i +"] [" +j +"] =");
                B[i][j] = leer.nextInt();
            }
        }
        System.out.println("\nLa matriz B es: ");
        for (i = 0; i<n; i++){
            for ( j= 0; j<n; j++){
                System.out.print(B[i][j] +"\t");
            }
            System.out.println("");
        }
        
        int [][] C = new int [n][n];
        for (i=0; i<n; i++){
            for (j = 0; j<n; j++){
                C[i][j] = A[i][j] + B [i][j];
            }
        }
        System.out.println("\nLa matriz C es: ");
        for (i = 0; i<n; i++){
            for ( j= 0; j<n; j++){
                System.out.print(C[i][j] +"\t");
            }
            System.out.println("");
        }
    }   
}
